lib/iex/lib/iex/helpers.ex

```diff
@@ -587,6 +588,25 @@ defmodule IEx.Helpers do
     raise ArgumentError, "import_file/1 expects a literal binary as its argument"
   end
 
+  @doc """
+  Calls `import/2` with the given arguments, but only if the module is available.
+
+  This lets you put imports in .iex.exs files (including ~/.iex.exs) without
+  getting compile errors if you open a console where the module is not available.
```

Comment:
We're missing a bunch of backticks for formatting here (`.iex.exs` and `~/.iex.exs`) :bowtie: